packages/desktop-client/src/components/budget/IncomeGroup.tsx

```diff
@@ -33,8 +44,18 @@ function IncomeGroup({
         onSave={onSave}
         onToggleCollapse={onToggleCollapse}
         onShowNewCategory={onShowNewCategory}
+        dragPreview={undefined}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind starting with lower-level components? Or doing multiple components at once?

We already have a bunch of these `undefined` prop values scattered throughout the codebase because of unfinished migrations. Ideally we should not add more of them.

Reply:
Yes I can do that. Thanks for the suggestion.